recipe_scrapers/__init__.py

```diff
@@ -1,4 +1,5 @@
 from __future__ import annotations
+from .ahealthysliceoflife import AHealthySliceOfLife
```

Comment:
```suggestion
```
While this is where the generate file placed this line it should be placed right after `from .aflavorjournal import AFlavorJournal` 

This is the result of a known bug reported in #1259